src/components/ReportActionItem/IOUPreview.js

```diff
@@ -224,7 +224,7 @@ const IOUPreview = (props) => {
                             </View>
                         )}
                     </View>
-
+                    <Text style={[styles.colorMuted]}>{Str.htmlDecode(lodashGet(props.action, 'originalMessage.comment', ''))}</Text>
```

Comment:
Lets remove this, we already add comment [here](https://github.com/Expensify/App/blob/d678e0e37b1e0b5f478002b9fcf68ac57c95f54c/src/components/ReportActionItem/IOUPreview.js#L233) 
```suggestion
```